### PR TITLE
[Feature] Update v1 normalizer tests

### PR DESCRIPTION
**Pull request by leb2dg**
* Rename test file
* Check that changesets were accepted

### CODE REVIEW

**Comment by chrisseto** [comment]
@leb2dg Could you update this when you have a moment?

**Comment by leb2dg** [comment]
@chrisseto, resolved conflicts

**Comment by leb2dg** [comment]
Tests are now broken though...will fix

File: api/views/workflow.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def post(self, request, *args, **kwargs):
         data = {}
         data['source'] = request.user.id
```
Comment (chrisseto) [inline api/views/workflow.py]: Could you remove this line? It's handled by passing request in (or at least should be)

**Pull request merged**